src/RendererPluginManager.php

```diff
@@ -42,28 +42,29 @@ class RendererPluginManager extends AbstractPluginManager
         'svg'   => Renderer\Svg::class,
 
         // Legacy Zend Framework aliases
-        \Zend\Barcode\Renderer\Image::class => Renderer\Image::class,
-        \Zend\Barcode\Renderer\Pdf::class => Renderer\Pdf::class,
-        \Zend\Barcode\Renderer\Svg::class => Renderer\Svg::class,
+        image::class => Renderer\Image::class,
+        pdf::class   => Renderer\Pdf::class,
+        svg::class   => Renderer\Svg::class,
 
         // v2 normalized FQCNs
         'zendbarcoderendererimage' => Renderer\Image::class,
-        'zendbarcoderendererpdf' => Renderer\Pdf::class,
-        'zendbarcoderenderersvg' => Renderer\Svg::class,
+        'zendbarcoderendererpdf'   => Renderer\Pdf::class,
+        'zendbarcoderenderersvg'   => Renderer\Svg::class,
     ];
 
+    /** @var array */
     protected $factories = [
         Renderer\Image::class => InvokableFactory::class,
         Renderer\Pdf::class   => InvokableFactory::class,
         Renderer\Svg::class   => InvokableFactory::class,
 
         // v2 canonical FQCNs
-
         'laminasbarcoderendererimage' => InvokableFactory::class,
         'laminasbarcoderendererpdf'   => InvokableFactory::class,
         'laminasbarcoderenderersvg'   => InvokableFactory::class,
     ];
 
+    /** @var string */
```

Comment:
Unsure if we can `{@inheritDoc}` this stuff perhaps? Just want to avoid diverging from parent class types (if they get upgraded)